### ISSUE TITLE
Header bar window buttons broken in the latest Firefox update

### ISSUE THREAD

**Issue by AthirsonSilva** [EliverLara/firefox-sweet-theme#85]
In the latest update of Firefox, the close, minimize and maximize window buttons are broken.

![Screenshot from 2023-05-10 08-56-24](https://github.com/EliverLara/firefox-sweet-theme/assets/84593887/97752220-05ed-46bc-9e98-e06b1ad6b61e)

This is the version of Firefox I am running on my machine, downloaded from Arch Linux repositories.

![Screenshot from 2023-05-10 09-00-27](https://github.com/EliverLara/firefox-sweet-theme/assets/84593887/26d85f17-3140-40dc-9567-6b15f9a6581a)

**Comment by ribru17** [comment]
To anyone else experiencing this, go to `~/.mozilla/firefox/\<your profile\>/chrome/firefox-sweet-theme/theme/parts/tabsbar.css and change:
```css
#TabsToolbar .titlebar-spacer {
	display: none !important;
}
```
to:
```css
#TabsToolbar .titlebar-spacer {
  width: 100px !important;
}
```

**Comment by EliverLara** [comment]
Sorry for the very late respopne, that was fixed now with 146442040be55fa654eee625ffe6626892bd3fb4
